ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -460,7 +460,8 @@ INDICATOR_NAMESPACES = {
         ('mvp_indicators', "MVP"),
     ),
 }
-COUCH_STALE_QUERY = 'ok'
+# should be 'ok' if couchdb version < 1.1.0, else 'update_after'
+COUCH_STALE_QUERY = 'update_after'
```

Comment:
like @snopoke said - we don't want to change this for prod/india so should be overrideable and the default should probably remain 'ok'